client/redux/actions/story.ts

```diff
@@ -0,0 +1,8 @@
+import { Story } from '~components/projects/types';
+
+export const setStory = (newStory: Story, storyPosition: number) =>
```

Comment:
Nit: I'd go with `story` over `newStory` as there's not a need to disambiguate the identifier and the "new" is implied by "setStory".